packages/app/src/runner/utils.ts

```diff
@@ -20,11 +20,7 @@ export function getReporterElement () {
 }
 
 export function empty (el: HTMLElement) {
-  while (el.lastChild) {
-    if (el && el.firstChild) {
-      el.removeChild(el.firstChild)
-    }
-  }
+  el.innerHTML = ''
```

Comment:
This function is called to empty the `#unified-runner` element. This is a parent div of the AUT iframe. At a cursory look, setting el.innerHTML at face value is described as less performant, but calling `removeChild` over and over causes recalculateStyles, which impacts performance.